.coveragerc

```diff
@@ -1,11 +1,22 @@
 [run]
-branch = True
+parallel = true
+source =
+    pyramid_openapi3
+;branch = True
```

Comment:
I like to have this as the reports are more detailed (it checks that all code branching is covered): https://coverage.readthedocs.io/en/v4.5.x/branch.html

Reply:
Done in cb5beaf